contrib/bdii/htcondor-ce-provider

```diff
@@ -123,10 +123,50 @@ def main():
 
     # Query collector for the number of CPUs and batch system Collector ad
     coll = htcondor.Collector()
-    total_cores = 0
-    for ad in coll.query(htcondor.AdTypes.Startd, 'State=!="Owner"', ['Cpus']):
-        if ad.get('Cpus'):
-            total_cores += ad['Cpus']
+    total_cores = {}
+    topologys = {}
+    #for ad in coll.query(htcondor.AdTypes.Startd, 'State=!="Owner" && PartitionableSlot =?= True', ['Arch', 'OpSys', 'OpSysMajorVer', 'OpSysName', 'DetectedCpus','DetectedMemory','HostGroup']):
+    for ad in coll.query(htcondor.AdTypes.Startd, 'State=!="Owner"', ['Arch', 'OpSys', 'OpSysMajorVer', 'OpSysName', 'DetectedCpus', 'DetectedMemory', 'Machine']):
+        if not ad['Machine'] in total_cores:
```

Comment:
Since you are using the Python 3 print function, might be worth (for consistency) to change all tests of the form `not x in y` to `x not in y` (see https://docs.python.org/3/reference/expressions.html#not-in)